web/pgadmin/browser/server_groups/servers/static/js/server.js

```diff
@@ -205,6 +205,12 @@ define('pgadmin.node.server', [
                   d = t.itemData(i);
                   t.removeIcon(i);
                   d.connected = false;
+
+                  // Generate the event that server is connected
```

Comment:
Please correct the tiny typo, connected should be disconnected.

Reply:
Done